src/globalmon/__init__.py

```diff
@@ -1 +1 @@
-# This file is important for setup.py to find modules.
\ No newline at end of file
+# This file is important for setup.py to find modules.
```

Comment:
no net to use this comment, anyone from python world should know the reason of this file

Reply:
I prefer code/type hints.. improves readability.